File: C++/Glicose.cpp
#include <iostream>
int main() {
    double glicose;
    std::cout << "Digite a quantidade de glicose no sangue: ";
    std::cin >> glicose;
    if (glicose <= 100.0) {
        std::cout << "Normal" << std::endl;
    } else if (glicose <= 140.0) {
        std::cout << "Elevado" << std::endl;
    } else {
        std::cout << "Diabetes" << std::endl;
    }
    return 0;
}

File: Glicose.cs
using System;

class Program {
    static void Main() {
        Console.Write("Quantidade de glicose: ");
        double glicose = double.Parse(Console.ReadLine());
        if (glicose <= 100)
            Console.WriteLine("Classificação: Normal");
        else if (glicose <= 140)
            Console.WriteLine("Classificação: Elevado");
        else
            Console.WriteLine("Classificação: Diabetes");
    }
}
